src/benchmarks/real-world/Roslyn/CompilerBenchmarks.csproj

```diff
@@ -15,7 +15,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.3.0" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.5.0" />
```

Comment:
This one is simple, but tricky. IIRC the main point of the Roslyn benchmarks was to see how runtime changes affect compilation time with Roslyn. That is why we have been using a fixed version of Roslyn (BTW we always wanted to add another set of CI legs that would use a fixed runtime and latest Roslyn).

Is this bump mandatory due to the BDN changes? If so, it's acceptable, we just need to remember that if there is a regression reported for these benchmarks it might be caused by the Roslyn change, not runtime change.

Reply:
I just noticed that the Roslyn dependency got downgraded to 4.1.0 in https://github.com/dotnet/BenchmarkDotNet/pull/2359 so this change should not be required anymore